galaxy.yml

```diff
@@ -26,7 +26,6 @@ build_ignore:
   - extensions
 dependencies:
   ansible.posix: ">=1.5.1"
-  fedora.linux_system_roles: ">=1.21.0"
```

Comment:
I am fine with merging, but since we are technically changing galaxy file should we bump versions?

Reply:
bump the version to 1.1.0!